### ISSUE TITLE
Respect browser font size settings in web app

### ISSUE THREAD

**Issue by Joelkw** [sourcegraph/sourcegraph#16602]
Until we have #120, this should be a good fix (especially for accessibility). This is requested by https://app.hubspot.com/contacts/2762526/company/693777200/ and is generally important for accessibility reasons. 

From @felixfbecker: 

> we should just respect the browser-wide font size setting (this shouldn't be a difficult fix, I think the problem is just that we have some absolute px font sizes in some places instead of always using relative `rem` or `em`).
